sky/clouds/aws.py

```diff
@@ -100,6 +100,14 @@ class AWSIdentityType(enum.Enum):
     #     region                us-east-1      config-file    ~/.aws/config
     SHARED_CREDENTIALS_FILE = 'shared-credentials-file'
 
+    def can_credential_expire(self) -> bool:
+        """Check if the AWS identity type can expire."""
+        expirable_types = {
+            AWSIdentityType.SSO, AWSIdentityType.ENV, AWSIdentityType.IAM_ROLE,
```

Comment:
Are IAM_ROLE and ENV guaranteed to be expirable? Is there some quick command we can run to check if credentials are expiring instead?

Reply:
I use ENV on my laptop, it is expirable. For IAM_ROLE need to check later.